### PR TITLE
expired link_invalid link page added

### PR DESCRIPTION
**Pull request by yash6102000**

### CODE REVIEW

**Review by NitinCopods — APPROVED** [review]

**Comment by anuragP04** [comment]
Test all the cases of appearance of these pages, getting wrong pages in end assesment and already submitted scenario

File: app/components/assessment/AlreadySubmitted.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  const { t } = useTranslation()
+  return (
+    <div className="flex h-screen flex-col bg-gray-50">
+      <Header />
+      <div className="flex-1 overflow-auto">
+        <div className="grid h-full items-center">
+          <div className="mx-auto flex w-coolDownCard flex-col items-center justify-center gap-10 bg-white py-16">
+            <img
+              src={alreadySubmitted}
+              alt="End test"
```
Comment (NitinCopods) [inline app/components/assessment/AlreadySubmitted.tsx:14]: use constant

File: app/components/assessment/EndAssessment.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const EndAssessment = () => {
       <Header />
       <div className="flex-1 overflow-auto">
         <div className="grid h-full items-center">
-          <div className="mx-auto flex w-coolDownCard flex-col items-center justify-center gap-8 bg-white py-16">
+          <div className="mx-auto flex w-coolDownCard flex-col items-center justify-center gap-10 bg-white py-16">
             <img
               src={EndAssessmentIcon}
               alt="End test"
```
Comment (NitinCopods) [inline app/components/assessment/EndAssessment.tsx:16]: use constant

File: app/models/sendgrid.servers.ts [hunk 3]
```diff
@@ -155,10 +155,12 @@ export async function sendTestInviteMail(email: string, link: string) {
   }
   await sendgrid.send(msg).then(
     () => {
+      console.log('sendd')
```
Comment (NitinCopods) [inline app/models/sendgrid.servers.ts:158]: remove console

File: app/models/sendgrid.servers.ts [hunk 4]
```diff
@@ -155,10 +155,12 @@ export async function sendTestInviteMail(email: string, link: string) {
...
       return 'ok'
     },
     (error) => {
       if (error.response) {
+        console.log(error)
```
Comment (NitinCopods) [inline app/models/sendgrid.servers.ts:163]: remove console

File: app/components/assessment/ExpiredLink.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  const { t } = useTranslation()
+  return (
+    <div className="flex h-screen flex-col bg-gray-50">
+      <Header />
+      <div className="flex-1 overflow-auto">
+        <div className="grid h-full items-center">
+          <div className="mx-auto flex w-coolDownCard flex-col items-center justify-center gap-10 bg-white py-16">
+            <img
+              src={InvalidLink}
+              alt="End test"
```
Comment (NitinCopods) [inline app/components/assessment/ExpiredLink.tsx:15]: use constant

**Pull request merged**